docs/source/topics/tf.rst

```diff
@@ -0,0 +1,128 @@
+Terraform Support
```

Comment:
We should probably specify what versions of Terraform are supportted here.

Reply:
at the moment support is pretty general, i've got a restriction in the generated output that says > 0.11.0 just for sanity which is basically any release in the last 18 months. i've done most of my testing with 0.11, but currently trying out 0.12.

Reply:
i've done enough apps/deploys with 0.12 to feel confident there as well. given terraform versions tends to more recent, i don't think we need an explicit doc here, the constraint in the generated output solves for accidental usage of truly ancient versions. if you feel differently let me know.